BlazorUI/Program.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace BlazorUI
+{
+    public class Program
+    {
+        public static void Main(string[] args)
```

Comment:
Codacy found an issue: [Make sure that command line arguments are used safely here.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5088867)